deno/v8/payloads/channel.ts

```diff
@@ -139,6 +139,12 @@ export enum ChannelType {
 	 * See https://discord.com/developers/docs/game-and-server-management/special-channels
 	 */
 	GUILD_STORE,
```

Comment:
Also, I think it would be worth explicitly setting values for the other channel types to avoid some accidental mistakes. It would also appear more consistent with the STAGE_CHANNEL line.

Reply:
Usually for enums, the only thing I do is set the value if it's a jump, and not explicitly incremental (see JSONErrorCodes)... It's simpler too :3